cook-county/cleaning-cook-county/extract_dataset.py

```diff
@@ -7,7 +7,7 @@
     parser = argparse.ArgumentParser()
     parser.add_argument('--steps_to_run', default='1,2,3,4,5', type=str)
     args = parser.parse_args()
-
+    os.environ['DATA_DIR'] = 'data_dir'
```

Comment:
You can add this as the default on the next line, but we shouldn't overwrite the user's environment here